src/main/CommandLine.cpp

```diff
@@ -1179,6 +1210,9 @@ runGenerateOrSimulateTxs(CommandLineArgs const& args, bool generate)
     // Default to no simulated transactions, just real data
     uint32_t multiplier = 1;
     bool verifyResults = false;
+#ifdef BEST_OFFER_DEBUGGING
```

Comment:
No need for runtime configuration in this path either.

Reply:
Done.